logging_config.py

```
@@ -0,0 +1,497 @@
+    
+    # Configure root logger
+    logging.basicConfig(
+        level=numeric_level,
+        handlers=handlers,
+        force=True
+    )
+    
+    # Log configuration
+    logger = logging.getLogger(__name__)
+    logger.info(
+        f"Logging configured",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)